docs/src/app/(public)/docs/[slug]/page.tsx

```diff
@@ -24,8 +24,10 @@ export default async function Docs({ params }: { params: { slug: string } }) {
   return (
     <div className="grid gap-6 grid-cols-[auto] md:grid-cols-[auto,12rem]">
       <div>
-        <h1 className="text-3xl font-bold sm:text-4xl mb-4">{page.title}</h1>
-        <div>
+        <h1 className="text-2xl font-extrabold sm:text-3xl mb-6">
+          {page.title}
+        </h1>
+        <div className="flex flex-col gap-4">
```

Comment:
I thought it makes sense to move this guy out to page.tsx, since this is where we control the layout, but shout if you don't agree @BarnabyBishop

Reply:
Good call